tutorials/retail_api_v2_update_product_java.md

```diff
@@ -0,0 +1,201 @@
+<walkthrough-metadata>
+  <meta name="title" content="Retail API. Update product tutorial" />
+  <meta name="description" content="Learn how to use Retail API Java library to update product tutorial" />
+  <meta name="component_id" content="593554" />
+  <meta name="keywords" content="retail, update product" />
+</walkthrough-metadata>
+
+# Update product tutorial
+
+## Get started
+
+To fill the catalog or to update a massive number of products, we recommend using the `import_products` method. However,
```

Comment:
let's remove all _ in method and field names, they are not used in Java for non final fields and method names

Reply:
done